deployment/pom.xml

```diff
@@ -9,11 +9,15 @@
   <artifactId>quarkus-asyncapi-deployment</artifactId>
   <name>Quarkus - AsyncAPI - Deployment</name>
   <dependencies>
-  <dependency>
+    <dependency>
       <groupId>io.quarkiverse.asyncapi</groupId>
       <artifactId>quarkus-asyncapi</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
```

Comment:
Is this really necessary?

Reply:
Lombok is used by the "official" async api library. 
I need to update the version to fix a bug